src/app/debug/filter-side-drawer/filter-side-drawer.component.ts

```diff
@@ -24,14 +24,14 @@ import { TitleCasePipe } from '@angular/common';
 })
 export class FilterSideDrawerComponent implements OnDestroy, OnInit {
   protected shouldShowFilter!: boolean;
-  protected metadataNames!: string[];
-  protected filters: Map<string, string> = new Map<string, string>();
+  protected metadataLabels!: string[];
   protected currentRecords: Map<string, Array<string>> = new Map<string, Array<string>>();
+  protected metadataTypes!: Map<string, string>;
 
-  shouldShowFilterSubscriber!: Subscription;
-  metadataNamesSubscriber!: Subscription;
-  filterSubscriber!: Subscription;
-  currentRecordsSubscriber!: Subscription;
+  shouldShowFilterSubscriber?: Subscription;
+  metadataLabelsSubscriber?: Subscription;
+  currentRecordsSubscriber?: Subscription;
+  metadataTypesSubscriber?: Subscription;
```

Comment:
Can you please call them subscription as that is exactly what they are, like the type suggests.